README.md

```diff
@@ -281,18 +281,14 @@ To preserve backward compatibility, we expect that `node file.js` will continue
 
   A package can be “dual mode” if its `package.json` contains both a `"main"` field and an `"exports"` field (or some other ESM-signifying field). An `import` statement of such a package will treat the package as ESM and ignore the `"main"` field. To explicitly import a dual-mode package via its CommonJS entry point, [`module.createRequireFromPath`][nodejs-docs-modules-create-require-from-path] could be used.
 
-  The ESM and CommonJS versions of a dual-mode package are really two distinct packages, and would be treated as such by Node if both were imported. It is an implementation detail to be worked out what Node should do in such a situation. Of particular concern are race conditions that could occur if it might be unpredictable which version (ESM or CommonJS) of a dual-mode package gets loaded first.
+  The ESM and CommonJS module systems have independent namespaces.  Therefore the ESM and CommonJS versions of a file have separate identities when instantiated.  The module loader's previous invariant was: a file will only be loaded at-most once within a Node application.  This invariant now becomes: a version of a file (ESM or CommonJS) will only be loaded at-most once within a Node application.
+  
+  Users that only rely on Node's interpretation of a module, via `import` and `import()`, will never trigger simultaneous instantiation of both versions.  Today, the only way to encounter this dual-instantiation scenario is if some part of the application uses `import`/`import()` to load a module **and** some other part of the application overrides Node's interpretation by using `require()` or  [`module.createRequireFromPath`][nodejs-docs-modules-create-require-from-path] to load that same module.
```

Comment:
I'm confused on if the usage of "today". Could we rephrase this and the above to be include phrasing like "with this proposal" and "without this proposal".